src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java

```diff
@@ -554,6 +554,21 @@ public static PersonalCircumstance aPersonalCircumstance() {
             .build();
     }
 
+    public static PersonalCircumstance aPersonalCircumstance(final LocalDate startDate, final LocalDate endDate) {
```

Comment:
there is no need to use final in parameter declaration. these are all effective final.

Reply:
More than happy to remove. I had included it as it looked like that was the pattern in the rest of the class